Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -260,6 +260,7 @@ private enum ValueMode
     private Unit _menuBlockOffset = Unit.Empty;
     private Unit _menuBlockItemOffset = Unit.Empty;
     private readonly DropDownMenu _optionsMenu;
+    private bool _enableTopLevelGrouping = false;
```

Comment:
Why do we have changes in BocList?
cc @a-ctor

Reply:
![image](https://user-images.githubusercontent.com/48289242/134636236-3e08c4fb-b2e2-4bfb-b956-49584f0d30d6.png)

Reply:
Ah, I forgot to ask that. But we have them because you wrote so in the ticket. I guess we can remove this :D

Reply:
@a-ctor
@derlistigelurch 
That was a spec-error. It should have been BocTreeView.